packages/cockpit/src/transactions/shared/export.js

```diff
@@ -0,0 +1,313 @@
+  path(['billing', 'address']),
+  path(['address'])
+)
+
+const getAddressSubProp = property =>
+  either(
+    pipe(
+      getAddressProp,
+      prop(property)
+    ),
+    () => LIMITER
+  )
+
+const getStatusName = pipe(
+  prop('status'),
+  prop(__, statusNames)
+)
+
+const isStatusRefused = propEq('status', 'refused')
+
+const getRefuseReason = pipe(
+  prop('status_reason'),
+  append(__, ['refused']),
+  path(__, statusNames)
+)
+
+const getStatus = ifElse(
+  isStatusRefused,
+  getRefuseReason,
+  getStatusName
+)
+
+const paymentMethodNames = {
+  boleto: 'Boleto',
+  credit_card: 'Cartão de Crédito',
+  default: '-',
+}
+
+const getPaymentMethod = pipe(
+  propOr('default', 'payment_method'),
+  prop(__, paymentMethodNames)
+)
+
+const completeNumberRegex = /(\+*[0-9]{2})([1-9]{2})(([9][0-9]{4})|[0-9]{4})([0-9]{4})/
+
+const isCompletePhoneNumber = number => completeNumberRegex.test(number)
+
+const formatPhoneNumber = (number) => {
+  if (!number) return ''
+  const phoneLength = length(number)
+  const [phoneRemainder, last4Digit] = splitAt((phoneLength - 4), number)
+
+  // this case is common for API versions 2013-03-01 and 2017-07-17
+  if (phoneLength <= 9) {
+    return `${phoneRemainder}-${last4Digit}`
+  }
+
+  const phoneRemainderLength = length(phoneRemainder)
+
+  if (isCompletePhoneNumber(number)) {
+    const splitIndex = phoneRemainderLength >= 10
+      ? phoneRemainderLength - 5
+      : phoneRemainderLength - 4
+    const [remainder, firstDigitis] = splitAt(
+      splitIndex,
+      phoneRemainder
+    )
+    const [ddi, ddd] = splitAt(length(remainder) - 2, remainder)
+    let outNumber = `${firstDigitis}-${last4Digit}`
+
+    if (ddd) {
+      outNumber = `(${ddd}) ${outNumber}`
+    }
+
+    if (ddi) {
+      outNumber = `${ddi} ${outNumber}`
+    }
+
+    return outNumber
+  }
+
+  const splitIndex = phoneRemainderLength >= 7
+    ? phoneRemainderLength - 5
+    : phoneRemainderLength - 4
+
+  const [remainder, firstDigitis] = splitAt(
+    splitIndex,
+    phoneRemainder
+  )
+
+  return `(${remainder}) ${firstDigitis}-${last4Digit}`
+}
+
+const formatPhone = pipe(
+  props(['ddd', 'number']),
+  juxt([
+    pipe(
+      head,
+      ddd => `(${ddd})`
+    ),
+    pipe(
+      last,
+      formatPhoneNumber
+    ),
+  ]),
+  join(' ')
+)
+
+const getRecipients = pipe(
+  prop('split_rules'),
+  ifElse(
+    is(Array),
+    pipe(
+      map(propOr('', 'id')),
+      join(', ')
+    ),
+    always('Recebedor Padrão')
+  )
+)
+
+const getPhone = pipe(
+  prop('phone'),
+  ifElse(
+    isEmptyOrNill,
+    always(LIMITER),
+    formatPhone
+  )
+)
+
+const formatPhones = pipe(
+  path(['customer', 'phone_numbers']),
+  ifElse(
+    isEmptyOrNill,
+    always(LIMITER),
+    pipe(
+      map(formatPhoneNumber),
+      join(', ')
+    )
+  )
+)
+
+const getPhones = ifElse(
+  pipe(
+    prop('phone'),
+    isEmptyOrNill
+  ),
+  formatPhones,
+  getPhone
+)
+
+const getId = ifElse(
+  pipe(
```

Comment:
esse pipe de condição do if pode ser:

```suggestion
 propSatisfies(isEmptyOrNill, 'tid'), 
```